docs/Software Architecture Document/Deployment View/assets/deployment_view.puml

```diff
@@ -0,0 +1,48 @@
+@startuml
+/'
+It is a source file for SDL SAD sequence diagram.
+For further information, please, follow http://plantuml.com/ 
+'/
+
+title Packages - Class Diagram
+
+cloud Cloud <<Clout service>> {
+frame Policies
+database Statistics
+}
+node MobileDevice <<device>> {
+  entity Application1
+  entity Application2
+  entity ApplicationN
+}
+
+
+node HeadUnit <<device>> {
+ folder SmartDeviceLink {
+    artifact "SDL binary"
+    artifact "Configuration Files"
+    artifact "Policy dynamic library"
```

Comment:
@LuxoftAKutsan  I'm not sure if this is applicable for OpenSDL but we don't have policy dynamic library in the pasa repo